spec/Records.tex

```diff
@@ -2,23 +2,25 @@
 \label{Records}
 \index{records}
 
-A record is a data structure that is similar to a class but instead has value
-semantics, similar to primitive types.  Value semantics mean that assignment, argument passing and function
-return values are by default all done by copying.  Value semantics also imply that a
-variable of record type is associated with only one piece of storage and has
-only one type throughout its lifetime.  Storage is allocated for a variable of
-record type when the variable declaration is executed, and the record variable
-is also initialized at that time.
-
-A record declaration statement creates a record type~\rsec{Record_Declarations}.  A
-variable of record type contains all and only the fields defined by that type
-(\rsec{Record_Types}).  Value semantics imply that the type of a record variable
-is known at compile time (i.e. it is statically typed).  
-
-A record can be created using the \chpl{new} operator, which
-allocates storage, initializes it via a call to a record constructor,
-and returns it.
-A record is also created upon a variable declaration of a record type.
+A record is a data structure that is similar to a class but instead has
+value semantics, similar to primitive types.  Value semantics mean that
+assignment, argument passing and function return values are by default
+all done by copying.  Value semantics also imply that a variable of
+record type is associated with only one piece of storage and has only one
+type throughout its lifetime.  Storage is allocated for a variable of
+record type when the variable declaration is executed, and the record
+variable is also initialized at that time.
```

Comment:
Let's add this at the end of the above paragraph:

When the record variable goes out of scope, or at the end of the program if it is a global, it is deinitialized and its storage is deallocated.